docs/smif-prerequisites.md

```diff
@@ -1,27 +1,341 @@
+```
+
+### Interventions
+
+Interventions represent the lowest level targets of decisions within an
+infrastructure simulation model.
+
+To enable the decision module of `smif`, interventions must be defined for each
+of the simulation models within a system-of-systems model.
+
+Once defined, these interventions can be exposed to various pieces of functionality
+within `smif`, including defining strategies and exploring decision space across
+infrastructure system-of-systems.
+
+| Attribute | Type | Example | Notes |
+| --- | --- | --- | --- |
+| name* | string | `nuclear_large_oxford` | Unique within a sector model |
+| capital_cost_value* | float | `20.3` ||
+| capital_cost_unit* | string | `£B` |
+| economic_lifetime* | integer | `20` | The duration over which the capital cost is ammortized|
+| operational_lifetime | integer | `25` | The duration during which the intervention is active |
+| location* | string | `Oxford` | |
+| capacity_value | float | `1000` | Example of a custom attribute |
+| capacity_unit | string | `MW` | Example of a custom attribute |
+| start_year | integer | `2018` | Example of a custom attribute |
+
+*required information
+
+### Narratives
+
+Narratives are the means by which packages of assumptions can be included in a
+model run. Narratives cut across model parameters, simulation models,
+and strategies, enabling complex combinations of user-defined functionality to
+be included in a model run.
+
+An example narratives file:
+
+```
+global:
+  discount_rate: 0.05
+energy_demand:
+  assump_diff_floorarea_pp: 0.5
+```
+
+### Scenarios
+
+This section is for [data providers](./smif-prerequisites.html#data-provider).
+
+Below is an example scenarios file which shows the change in population
+for three regions, England, Scotland and Wales for the years
+2010, 2015 and 2020.
+The polygons associated with the regions are stored in a region definitions
+file.
+The duration associated with the interval `1` are stored
+in the interval definitions file.
+See the [metadata](./smif-prerequisites.html#metadata) section for details on
+how to define region and interval definitions if you have not already done so.
+
+```
+timestep,region,interval,value
+2010,England,1,52000000
+2010,Scotland,1,5100000
+2010,Wales,1,2900000
+2015,England,1,53000000
+2015,Scotland,1,5300000
+2015,Wales,1,3000000
+2020,England,1,54000000
+2020,Scotland,1,5500000
+2020,Wales,1,3200000
+```
+
+The entries in the `timestep` column should be a valid year integer.
+
+The entries in the `region` column should refer to the names of the regions
+in the associated [region definition file](./smif-prerequisites.html#regions).
+
+The entries in the `interval` column should refer to the id of the intervals in
+the associated [interval definition file](./smif-prerequisites.html#intervals).
+
+The entries in the `value` column will normally be a floating point number.
+
+The metadata required to define a particular scenario are shown in the table
```

Comment:
I would move all of the metadata information out of this chapter, it is confusing. This chapter defines the data requirements. The metadata becomes important during the project configuration.